package.json

```diff
@@ -31,17 +31,16 @@
     "babel-core": "^6.16.0",
     "babel-plugin-transform-es2015-destructuring": "^6.9.0",
     "coveralls": "^2.11.9",
-    "eslint": "^2.4.0",
-    "eslint-config-google": "^0.4.0",
-    "google-closure-compiler": "^20160517.0.0",
+    "eslint": "^3.12.0",
+    "eslint-config-google": "^0.7.1",
+    "google-closure-compiler": "^20160911.0.0",
```

Comment:
any reason not to use the latest from 12/1?
https://github.com/google/closure-compiler/wiki/Releases#december-1-2016-v20161201